tests/UIButtons.js

```diff
@@ -0,0 +1,173 @@
+      console.log(color);
+      const valueColor = color.parsed.hex;
+      await expect(valueColor).toEqual("#898989");
+
+      const alignItems = await elem.getCSSProperty("align-items");
+      console.log(alignItems);
+      const valueAlignItems = alignItems.value;
+      await expect(valueAlignItems).toEqual("center");
+    }
+  });
+
+  it("Checking the appearance of the submenu buttons", async () => {
+    const submenuButtons = [
+      await Menu.aboutButton,
+      Menu.privacyPolicyButton,
+      Menu.dmcaButton,
+      Menu.advertisingButton,
+    ];
+
+    await (await Menu.moreButton).waitForDisplayed();
+    await (await Menu.moreButton).click();
+    await browser.pause(5000);
+
+    for (let i = 0; i < submenuButtons.length; i++) {
+      const object = await submenuButtons[i];
+
+      await browser.pause(5000);
+
+      let fontSize = await object.getCSSProperty("font-size");
+      console.log(fontSize);
+      const valueFontSize = await fontSize.value;
+      await expect(valueFontSize).toEqual("16px");
+
+      const color = await object.getCSSProperty("color");
+      console.log(color);
+      const valueColor = await color.parsed.hex;
+      await expect(valueColor).toEqual("#898989");
+
+      const pandding = await object.getCSSProperty("padding-right");
+      console.log(pandding);
+      const valuePandding = await pandding.value;
+      await expect(valuePandding).toEqual("10px");
+
+      const width = await object.getCSSProperty("width");
+      console.log(width);
+      const valueWidth = await width.value;
+      await expect(valueWidth).toEqual("150px");
+    }
+  });
+
+  it("Checking the appearance of the search button", async () => {
+    const search = await Menu.searchButton;
+
+    const fontSize = await search.getCSSProperty("font-size");
+    console.log(fontSize);
+    const valueFontSize = await fontSize.value;
+    await expect(valueFontSize).toEqual("16px");
+
+    const paddingLeft = await search.getCSSProperty("padding-left");
+    console.log(paddingLeft);
+    const valuePaddingLeft = await paddingLeft.value;
+    await expect(valuePaddingLeft).toEqual("15px");
+  });
+
+  it("Checking the appearance of the search icon", async () => {
+    const searchIcon = await Menu.searchIcon;
+
+    const backgroundPosition = await searchIcon.getCSSProperty(
+      "background-position"
+    );
+    console.log(backgroundPosition);
+    const valueBackgroundPosition = await backgroundPosition.value;
+    await expect(valueBackgroundPosition).toEqual("-290px -129px");
+
+    const width = await searchIcon.getCSSProperty("width");
+    console.log(width);
+    const valueWidth = await width.value;
+    await expect(valueWidth).toEqual("20px");
+
+    const height = await searchIcon.getCSSProperty("height");
+    console.log(height);
+    const valueHeight = await height.value;
+    await expect(valueHeight).toEqual("20px");
+  });
+
+  it("Checking the appearance of the see more buttons", async () => {
```

Comment:
I think you could specify here that it is checking the style for More menu from the header.